.buildkite/pipeline.yml

```diff
@@ -29,7 +29,7 @@ steps:
       - "$SVERDRUP_HOME/julia-$JULIA_VERSION/bin/julia -O0 --color=yes --project -e 'using Pkg; Pkg.test()'"
     agents:
       queue: Oceananigans
-      architecture: GPU
+      architecture: CPU
```

Comment:
@ali-ramadhan, why this became `CPU`? Is this some leftover from debugging?

Reply:
Ah good catch! I forgot to switch back to using Sverdrup. Just added a commit to switch back.